flashCards.rb

```diff
@@ -0,0 +1,123 @@
+require "pry"
+class Flashcard 
+	attr_accessor :front, :back
+	def initialize
+
+	@front = "front"
+    @back = "back"
+    @new_front = []
+    @new_back = []
+
+    @@default_cards = [@front, @back]
+    @@new_cards = [@new_front, @new_back]
+  end
+
+	def create_flash new_front, new_back
+		@new_front << new_front
+		@new_back << new_back
+	end
+
+		def self.current_card 
+			puts "Current Flash Cards"
+			
+			puts "Default game cards #{@@default_cards}"
+
+			puts "Player made set #{@@new_cards}"
+		end
+
+		def edit_flash
+
+		puts "Choose if you would like to edit your front or back cards. Type \"back\" or \"front\""
+		edit_input = gets.chomp
+		card_index = []
+		to_change = []
+		if edit_input == "front"
+			@@new_cards[0].each_with_index {
+				|card, index|
+				hash[card] = index
+				card_index = gets.chomp
+			}
+			hash
+			puts "Select the card by number."
+			editing = gets.to_i
+			case editing
+			when editing == (0..hash)
+				to_change <<
+				[to_change[0]] << gets.to_s
+			end
+
+		elsif edit_input == "back"
+			@@new_cards[1].each_with_index {
+				|card, index|
+				hash[card] = index
+				card_index = gets.chomp
+			}
+			hash 
+			puts "Select the card by number."
+			editing = gets.to_i
+			case editing
+			when editing == (0..hash)
+				to_change <<
+				[to_change[0]] << gets.to_s
+			end
+
```

Comment:
With the suggestion above, you can use `@@new_cards.each` to iterate over all the existing flash cards and prompt the user to edit one. You could use a while loop to capture only valid input, see the ruby enums lesson for applicable code snippets using `while` and `until`.